server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/main/java/org/apache/james/jmap/methods/integration/JamesWithSpamAssassin.java

```diff
@@ -0,0 +1,41 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+package org.apache.james.jmap.methods.integration;
+
+import org.apache.james.GuiceJamesServer;
+import org.apache.james.spamassassin.SpamAssassinExtension;
+
+public class JamesWithSpamAssassin {
```

Comment:
I remember I had a [PR](https://github.com/linagora/james-project/pull/1938/files) to remove `JamesWithSpamAssassin`, why do you re-create it?

Reply:
Sorry, the cherry-pick brings it back.
I will remove it.